src/redux/reducers/initialState.js

```diff
@@ -1,186 +1,40 @@
 import modalTypes from '../../enums/modalTypes';
 
 export default {
```

Comment:
I think we should still have the dummy data as initial state especially for features that we have not completed yet

Reply:
for the features that ain't done we can add the dummy data but for the completed ones I don't think we need the dummy data

Reply:
I restored the dummy data for the incomplete features.